packages/sdk/src/swap/sdk.ts

```diff
@@ -43,6 +44,8 @@ export class SwapSDK {
 
   private readonly brokerConfig?;
 
+  private environment?: Environment;
```

Comment:
not sure if i like the name environment here. I think it can be something more unique to this situation. i.e `stateChainEnvironment` or  for something shorter `constants`

Reply:
i like stateChainEnvironment